contrib/python/api/tests/tests.py

```diff
@@ -216,10 +218,30 @@ def test_topology_rules(self):
         node2 = restclient.lookup_nodes("G.V().Has('Name', 'node2')")
         self.assertEqual(len(node2), 1, "should find only one node as node2")
 
+
         edge = restclient.lookup_edges(
                 "G.E().Has('RelationType', 'layer2', 'EdgeName', 'my_edge')")
         self.assertEqual(len(edge), 1, "should find only one edge")
 
-        restclient.edgerule_delete(edgerule.uuid)
+        noderules = restclient.noderule_list()
+        self.assertGreaterEqual(len(noderules), 1, "no noderules found")
+        for noderule in noderules:
+            if (noderule.uuid == noderule1.uuid):
+                found = True
+                break
+
+        self.assertTrue(found, "created noderule not found")
+
+        edgerules = restclient.edgerule_list()
+        self.assertGreaterEqual(len(edgerules), 1, "no edgerules found")
+        for edgerule in edgerules:
```

Comment:
add "found = False" before the loop as you are on the context (function)

Reply:
> add "found = False" before the loop as you are on the context (function)

Done